motioneye/config.py

```diff
@@ -915,7 +915,7 @@ def motion_camera_ui_to_dict(ui, prev_config=None):
         'threshold_maximum': ui['max_frame_change_threshold'],
         'threshold_tune': ui['auto_threshold_tuning'],
         'noise_tune': ui['auto_noise_detect'],
-        'noise_level': max(1, int(round(int(ui['noise_level']) * 2.55))),
+        'noise_level': max(1, round(ui['noise_level'] * 2.55)),
```

Comment:
Without the inner `int` there's `TypeError: can't multiply sequence by non-int of type 'float'`

Reply:
Ah, so the UI is generating numbers in string format, and this is hence the reason to convert them to int first.